recipes/ipypdf/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: ipypdf
+  version: {{ version }}
+
+source:
+  url: https://github.com/joelstansbury/ipypdf/archive/0206fb7.tar.gz
+  sha256: ec1e9cfd2db9e45963e2664d86acfdaf7b5c4568e654ceb3c6f60d7a29f65f3d
+
+build:
+  script: {{ PYTHON }} -m pip install -vv . --no-deps
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - pip
+    - ipycanvas
+    - ipycytoscape
+    - ipyevents
+    - ipytree
+    - ipywidgets
+    - jupyterlab >=3.0
+    - numpy <=1.19.3,>=1.13  # Constrained by paddlepaddle
+    - pandas
+    - pdf2image
+    - poppler
+    - pytesseract
+    - spacy
+    - spacy-model-en_core_web_lg
+    - tesseract
+    - traitlets
+
+test:
+  source_files:
+    - tests
+  requires:
+    - pytest
+    - pytest-cov
+  commands:
+    - pytest
+
+about:
+  home: https://github.com/JoelStansbury/ipypdf
+  summary: Jupyter widget for applying nlp to pdf documents
+  license: MIT
+  license_file: LICENSE
+  description: |
+    ipypdf is an InteractivePython (iPy) widget designed to provide powerful
+    tools for extracting data from PDF documents. The widget features a directory
+    tree to navigate through the hierarchy. Rendered pdf canvas used to highlight
+    portions of the document relevant to your specific task. And a toolkit panel
+    to perform predefined operations. Among these operations are AutoParse, which
+    will utilize layoutparser to extract titles, text, tables, and images from the
+    document. Spacy, which utilizes SpaCy's NLP model to detect and display named
+    entities. Cytoscape, which discovers symantically similar portions of text and
+    displays them in an interactive graph. (and more)
+  doc_url: https://github.com/JoelStansbury/ipypdf
```

Comment:
while metadata are nice, it doesn't buy much if `home`, `docs` and `dev` are all the same thing.